TAInformation/views.py

```diff
@@ -63,34 +63,36 @@ def get(self, request):
         return render(request, "createCourse.html", {})
 
     def post(self, request):
-        noPermissions = canAccess(User.objects.get('role'), 1)
-
+        noPermissions = canAccess(1, 1) #User.objects.get('role')
```

Comment:
I think using User.objects.get('role').role might be better than hardcoding a value in

Reply:
Fixed